packages/flutter/lib/src/foundation/diagnostics.dart

```diff
@@ -1600,29 +1829,31 @@ class DiagnosticsProperty<T> extends DiagnosticsNode {
     }
   }
 
-  /// If the [value] of the property equals [defaultValue] the property is
-  /// [hidden] as the [value] is uninteresting.
+  /// If the [value] of the property equals [defaultValue] the priority [level]
+  /// of the property is downgraded to [DiagnosticLevel.fine] as the property
+  /// value is uninteresting.
```

Comment:
these docs should talk about `missingIfNull`

Reply:
done